### ISSUE TITLE
Support f:passThroughAttribute

### ISSUE THREAD

**Issue by Rapster** [primefaces/primefaces#2595]
## 1) Environment
- PrimeFaces version: 6.1
- Does it work on the newest released PrimeFaces version? Version?
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source)
- Application server + version:
- Affected browsers:

## 2) Expected behavior
Should encode "<f:passThroughAttribute />"
## 3) Actual behavior
Encode "<f:passThroughAttributes" but not "<f:passThroughAttribute />"

## 4) Steps to reproduce
```xml
<p:fileUpload>
    <f:passThroughAttribute name="x" value="y"/>
<p:fileUpload>
```

## 5) Tips
```java
 public static void renderPassThroughAttributes(FacesContext context, UIComponent component) throws IOException {
        ResponseWriter writer = context.getResponseWriter();
        Map<String, Object> passthroughAttributes = component.getPassThroughAttributes(false);
       ...
```

Basically the method read "PropertyKeys.passThroughAttributes"

**Comment by tandraschko** [comment]
PR is welcome ;)

**Comment by tandraschko** [comment]
Works fine for me.

**Comment by Rapster** [comment]
Okay strange... I may have done something wrong then
